source/boards/working-with-boards.rst

```diff
@@ -52,4 +74,48 @@ Table views
 
 Each column corresponds to a card property. You can edit cells directly or you can select **Open** to open the card editor for that row.
 
-Select the headers to sort them or insert new properties.
+=======
+
+All changes you make to boards and cards are saved immediately.
+
+Changing views
+--------------
+
+For this exercise, select the **Project Tasks** template. The first view of the new board is a table of all tasks.
+
+* Select **By Status** in the sidebar to see a board view.
+* Select **Properties** and enable **Priority** and **Date created** to add those properties to the card display.
+* You can also change the **Group By**, **Filter**, and **Sort** settings of the view.
+
+Adding cards
+------------
+
+Select **New** to add a new card to a board.
+
+Editing cards 
+-------------
+
+Select a card to edit it. A card consists of:
+
+* **A set of properties:** Properties are common to all cards in a board. Board views can group cards by “Select” type properties into different columns.
```

Comment:
Should we also call out that a card has:
* A title
* An icon / emoji - to quickly distinguish it

Not sure if this is needed, but can be added later if you think so.

Reply:
I think we should, definitely! Thanks!